src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -28,11 +30,23 @@ const NavHeader = (props) => {
     isHeaderHide,
     topicDetail,
     setIsFollow,
-    isFollow
+    isFollow,
+    hasSearch,
+    searchText,
+    setSearchText,
+    setDiscoveryLoadingData,
+    isFocus,
+    setIsFocus,
+    fetchDiscoveryData,
+    onCancelToken,
+    placeholderText,
+    setIsFirstTimeOpen,
+    getSearchLayout
   } = props;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Multiple new props are being destructured from `props`. Make sure these props are passed correctly where this component is used.
```diff
+     isFollow,
+     hasSearch,
+     searchText,
+     setSearchText,
+     setDiscoveryLoadingData,
+     isFocus,
+     setIsFocus,
+     fetchDiscoveryData,
+     onCancelToken,
+     placeholderText,
+     setIsFirstTimeOpen,
+     getSearchLayout
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->